src/codecatalyst/activation.ts

```diff
@@ -35,6 +35,8 @@ export async function activate(ctx: ExtContext): Promise<void> {
     const commands = new CodeCatalystCommands(authProvider)
     const remoteSourceProvider = new CodeCatalystRemoteSourceProvider(commands, authProvider)
 
+    await authProvider.restore()
```

Comment:
which commit (accidentally) removed this? is likely worth reviewing it

Reply:
Yup. Are we sure that this was actually here before?

Reply:
no but it used to autoconnect